app/views/admin/radios/index.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.array! @radios, partial: 'radios/radio', as: :radio
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [SideCI](https://sideci.com) -->